tests/integration/commontestutils/skrcontextimpl/dual_cluster.go

```diff
@@ -137,7 +142,6 @@ func (f *DualClusterFactory) Stop() error {
 			}
 		}
 		f.SkrEnvs.Delete(key)
-		f.clients.Delete(name)
```

Comment:
This doesn't work anymore since we don't have the NamespacedName available. Providing it to here would also be a not entirely trivial thing.

I opted to remove and not delete it as it is anyway in memory and will be cleaned up by the garbage collection. The important thin gis that we call `stopper.Stop()` above to kill etcd/kube-apiserver processes on the machine (which is anyway not working probably currently...)